packages/syntax/src/Syntax.story.tsx

```diff
@@ -28,6 +28,15 @@ storiesOf('Syntax', module).add(
         showLineNumbers={boolean('Show line numbers', false)}
         darkMode={boolean('darkMode', false)}
         language={select('language', Language, Language.JavaScript)}
+        highlightLines={select(
+          'highlight lines',
+          {
+            none: undefined,
+            single: [1],
+            multiple: [2, 3, 5],
```

Comment:
can we have a range example here too

Reply:
Doing so works, but causes a TS error, presumably because of the typing of the knobs ☹️

Reply:
you could pull the values out of the knob, e.g.
```ts
const examples = {
  none: undefined,
  single: [1],
  multiple: [2, 3, 5],
};

highlightLines={
  examples[
    select('highlight lines', Object.keys(examples), 'none')
  ]
}
```

Reply:
Smart